packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -311,10 +311,8 @@ export default class QuanticSearchBoxInput extends LightningElement {
 
   clearInput() {
     this.sendInputValueChangeEvent('');
+    this.input.value = '';
```

Comment:
Agreed, just change that and it's good to go for me :)

Reply:
changed that and also added a small fix for the weird behaviour where it would collapse and then clear. It was happening because the on blur was called before the onclick and therefore calling onBlur (where we collapse the searchbox). I added a mousedown on the button that preventDefault and therefore the blur.

proof it works:

https://github.com/user-attachments/assets/c2963afb-0cf3-4e36-b8fe-5238f498a5e2